src/ui/events.js

```diff
@@ -192,7 +194,7 @@ export class MapTouchEvent extends Event {
  */
 export class MapWheelEvent extends Event {
     /**
-     * The event type.
+     * The event type (one of {@link Map.event:wheel}).
```

Comment:
```suggestion
     * The event type ({@link Map.event:wheel}).
```
> I don't think the "one of..." construction works here
^ Agreed! Corrected.